src/api/app/views/webui/request/show.html.erb

```diff
@@ -2,6 +2,10 @@
 <% @layouttype = 'custom' %>
 <% @crumb_list = ["Request #{@bs_request.number}"] %>
 
+<% content_for(:meta_title, "#{@pagetitle} (#{@bs_request.state})") %>
+<% content_for(:meta_description, @bs_request.description) %>
+<% content_for(:meta_image, gravatar_icon((User.find_by_login(@bs_request.creator).email), 500)) %>
```

Comment:
500px are a lot - is that normal for meta_image?

Reply:
I originally assumed that a bigger image would be best due to wherever these links get shared, the image will be resized to the site/app's open graph styling.

I tried to test this after reading your comment and found the picture wasn't loading at all, Oops! I ended up not using `gravatar_icon` and using the default Gravatar image size.

I will write some more about this below.